.build/README.md

```
@@ -2,15 +1,18 @@
+# .build scripts
 
-This document describes the script used to dynamically determine whether DSC
-resource integration tests should run in Azure Pipelines.
+Documentation for the SqlServerDsc module build/pipeline scripts.
 
-## What the Script Does
+## `Test-ShouldRunDscResourceIntegrationTests.ps1`
+
+This script dynamically determines whether DSC resource integration tests should run in Azure Pipelines.
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 104

(MD013, line-length)